app/packages/core/src/components/ColorModal/LabelTag.tsx

```diff
@@ -0,0 +1,235 @@
+import { isValidColor } from "@fiftyone/looker/src/overlays/util";
+import * as fos from "@fiftyone/state";
+import { getColor } from "@fiftyone/utilities";
+import { Divider } from "@mui/material";
+import colorString from "color-string";
+import _, { cloneDeep } from "lodash";
+import React, { useCallback, useEffect, useState } from "react";
+import { TwitterPicker } from "react-color";
+import { useRecoilValue } from "recoil";
+import Checkbox from "../Common/Checkbox";
+import Input from "../Common/Input";
+import { resetColor } from "./ColorFooter";
+import {
+  FieldCHILD_STYLE,
+  FieldColorSquare,
+  PickerWrapper,
+  SectionWrapper,
+} from "./ShareStyledDiv";
+import AttributeColorSetting from "./colorPalette/AttributeColorSetting";
+import { colorPicker } from "./colorPalette/Colorpicker.module.css";
+import ModeControl from "./controls/ModeControl";
+
+type State = {
+  useLabelColors: boolean;
+  useFieldColor: boolean;
+};
+
+const LabelTag: React.FC = () => {
+  const wrapperRef: React.RefObject<HTMLDivElement> = React.useRef();
+  const pickerRef: React.RefObject<TwitterPicker> = React.useRef();
+  const { colorPool, fields, labelTags } = useRecoilValue(
+    fos.sessionColorScheme
+  );
+
+  const setColorScheme = fos.useSetSessionColorScheme();
+  const coloring = useRecoilValue(fos.coloring(false));
+  const color = getColor(colorPool, coloring.seed, "_label_tags");
+
+  const [showFieldPicker, setShowFieldPicker] = useState(false);
+  const [input, setInput] = useState(color);
+  const [colors, setColors] = useState(colorPool);
+  const [state, setState] = useState<State>({
+    useLabelColors: Boolean(
+      labelTags?.valueColors && labelTags.valueColors.length > 0
+    ),
+    useFieldColor: Boolean(labelTags?.fieldColor),
+  });
```

Comment:
If a session update were to come from the server while this component is shown, this state would not update I believe. It is an edge case, but it'd be good consider those situations (not super important now, but we should think about situations like that when adding test coverage)